input/data/tasks/herd_init_task.json

```
@@ -0,0 +1,13 @@
+{
+  "parallel_workers": 4,
+  "tasks": [
+    {
+      "task_type": "HERD_INITIALIZATION",
+      "metadata_file_path": "input/metadata/default_metadata.json",
+      "output_prefix": "herd_init",
+      "save_animals": true,
+      "log_verbosity": "logs",
```

Since logs will print out a lot of unnecessary warnings in addition to the logs, I think this would be better left at errors unless you have a strong reason otherwise.

```suggestion
      "log_verbosity": "errors",
```

---

Nope, no strong reason, I just like having a big wall of colorful text to read when I run RuFaS